data/human/deep missions.txt

```diff
@@ -4058,7 +4058,7 @@ mission "Hjlod Remembers Windblain"
 	on offer
 		dialog
 			`Your inbox hums with a new message from <planet>:`
-			`	"Dear <first>,`
+			`	"Dear <first>,"`
 			`	I never properly thanked you for saving my life on Nifel all those years ago. If you're ever in the neighborhood, please stop by <planet>.`
 			`				Signed, Hjlod"`
```

Comment:
Ditto.

Reply:
Shouldn't `"Dear <first>,"` loose it's indentation?